### ISSUE TITLE
KeyError: WERKZEUG_SERVER_FD

### ISSUE THREAD

**Issue by jjbrosnan** [bluenote10/tabloo#21]
I'm trying to use Tabloo from a Jupyter Notebook.  When trying to use it to view a Pandas DataFrame, I get this stacktrace:

```
'FLASK_ENV' is deprecated and will not be used in Flask 2.3. Use 'FLASK_DEBUG' instead.
'FLASK_ENV' is deprecated and will not be used in Flask 2.3. Use 'FLASK_DEBUG' instead.
---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
Input In [9], in <module>
----> 1 tabloo.show(df_bikeride)

File ~/miniconda3/lib/python3.9/site-packages/tabloo/tabloo.py:33, in show(df, open_browser, server_port, server_logging, debug)
     10 def show(
     11         df,
     12         open_browser=True,
   (...)
     15         debug=False,
     16     ):
     17     """
     18     Runs a Tabloo app on a given dataframe.
     19 
   (...)
     31     :return:
     32     """
---> 33     server.serve(
     34         df=df,
     35         open_browser=open_browser,
     36         server_port=server_port,
     37         server_logging=server_logging,
     38         debug=debug,
     39     )

File ~/miniconda3/lib/python3.9/site-packages/tabloo/server.py:128, in serve(df, open_browser, server_port, server_logging, debug)
    125 if open_browser:
    126     threading.Timer(0.25, lambda: webbrowser.open(url)).start()
--> 128 app.run(
    129     port=server_port,
    130     debug=debug,
    131     use_reloader=debug,
    132     processes=1,
    133     threaded=False,
    134 )

File ~/miniconda3/lib/python3.9/site-packages/flask/app.py:1188, in Flask.run(self, host, port, debug, load_dotenv, **options)
   1185 from werkzeug.serving import run_simple
   1187 try:
-> 1188     run_simple(t.cast(str, host), port, self, **options)
   1189 finally:
   1190     # reset the first request information if the development server
   1191     # reset normally.  This makes it possible to restart the server
   1192     # without reloader and that stuff from an interactive shell.
   1193     self._got_first_request = False

File ~/miniconda3/lib/python3.9/site-packages/werkzeug/serving.py:1069, in run_simple(hostname, port, application, use_reloader, use_debugger, use_evalex, extra_files, exclude_patterns, reloader_interval, reloader_type, threaded, processes, request_handler, static_files, passthrough_errors, ssl_context)
   1067     os.environ["WERKZEUG_SERVER_FD"] = str(fd)
   1068 else:
-> 1069     fd = int(os.environ["WERKZEUG_SERVER_FD"])
   1071 srv = make_server(
   1072     hostname,
   1073     port,
   (...)
   1080     fd=fd,
   1081 )
   1083 if not is_running_from_reloader():

File ~/miniconda3/lib/python3.9/os.py:679, in _Environ.__getitem__(self, key)
    676     value = self._data[self.encodekey(key)]
    677 except KeyError:
    678     # raise KeyError with the original key value
--> 679     raise KeyError(key) from None
    680 return self.decodevalue(value)

KeyError: 'WERKZEUG_SERVER_FD'
```

I haven't been able to find any `werkzeug` dependency in tabloo.  I'm using version 2.2.2.

**Comment by jjbrosnan** [comment]
I was able to recreate this error with another library, so it's something on my end.  I'm going to try to downgrade to an earlier version of werkzeug to see if that will work.

**Comment by bluenote10** [comment]
> I haven't been able to find any werkzeug dependency in tabloo. I'm using version 2.2.2.

`werkzeug` should be a transitive dependency of flask. Perhaps the flask <-> werkzeug dependency is somehow broken?

**Comment by jjbrosnan** [comment]
I tried downgrading to werkzeug 2.0.0 only to encounter a new problem:

```
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
Input In [1], in <module>
      3 from pandasgui import show
      4 import pandas as pd
----> 5 import tabloo
      6 import dtale
      7 import qgrid

File ~/miniconda3/lib/python3.9/site-packages/tabloo/__init__.py:1, in <module>
----> 1 from .tabloo import (
      2     show, embedHTML,
      3 )

File ~/miniconda3/lib/python3.9/site-packages/tabloo/tabloo.py:7, in <module>
      4 import json
      5 import pandas as pd
----> 7 from . import server
     10 def show(
     11         df,
     12         open_browser=True,
   (...)
     15         debug=False,
     16     ):
     17     """
     18     Runs a Tabloo app on a given dataframe.
     19 
   (...)
     31     :return:
     32     """

File ~/miniconda3/lib/python3.9/site-packages/tabloo/server.py:8, in <module>
      5 import threading
      6 import webbrowser
----> 8 from flask import Flask, send_from_directory, redirect, request, make_response
      9 from flask_cors import CORS
     11 from .backend import Backend, to_json

File ~/miniconda3/lib/python3.9/site-packages/flask/__init__.py:4, in <module>
      1 from markupsafe import escape
      2 from markupsafe import Markup
----> 4 from . import json as json
      5 from .app import Flask as Flask
      6 from .app import Request as Request

File ~/miniconda3/lib/python3.9/site-packages/flask/json/__init__.py:8, in <module>
      4 import typing as t
      6 from jinja2.utils import htmlsafe_json_dumps as _jinja_htmlsafe_dumps
----> 8 from ..globals import current_app
      9 from .provider import _default
     11 if t.TYPE_CHECKING:  # pragma: no cover

File ~/miniconda3/lib/python3.9/site-packages/flask/globals.py:4, in <module>
      1 import typing as t
      2 from contextvars import ContextVar
----> 4 from werkzeug.local import LocalProxy
      6 if t.TYPE_CHECKING:  # pragma: no cover
      7     from .app import Flask

ModuleNotFoundError: No module named 'werkzeug.local'
```

The only similar problem I'm able to find is in [this GitHub issue thread](https://github.com/cs01/gdbgui/issues/425).  User @davidism suggests commenting out a line in that thread that sets a variable based off an environment variable.  I know very little about werkzeug so I have no idea if that thread is relevant to this issue.

**Comment by bluenote10** [comment]
Sorry for the delay ;) this should be fixed in version 0.1.0.
